projects/managed-hubs/support.md

```diff
@@ -173,24 +173,62 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
-4. **Communicate status**. Once we have an issue created to track the next steps, send a message to the Community Representative letting them know about the situation and what the next steps will be. Here's a template to help guide you:
+The goal of the non-incident response process is to bring standardization to our support response. This simple workflow tries to battle the bias towards a reactive response whereas it is also bringing some common patterns so all of our non-incident support responses are cohesive and shared among our support stewards.
+The current iteration of the workflow states each step and who should be responsible/accountable for the specific step, plus some other clarifications.
+
+When a new ticket lands in Freshdesk under the support group and it is not an incident, you should follow the following steps:
+
+1. `Who: Support steward`
+
+   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know about any investigation we have done thus far.
+2. `Who: Support steward`
+
+   **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
+   1. If you resolve the issue, then jump to the "Confirm resolution" step 7.
+   2. If you don't believe you can resolve the issue (or you couldn't) in 30 minutes, jump to the next step.
+3. `Who: Support Steward, the board metadata will be revisited by a Partnerships representative and the Engineering Manager`
+
+   **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and/or you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
+
+   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. It should be automatically added to the **Eng & Prod** board by the existing automation. Add the **type**: `support` and choose an **impact** level in the **Eng & Prod** board metadata (available for the very same issue, no need to visit the board itself!).
+   
+   If the issue has a `critical` impact (we defer that first evaluation to the support steward), an additional ping to the support Slack channel is needed to boost the signal.
+   
+   If the support steward has the capacity, they should self-assign the `critical` issue and work on it (this is now outside of the 30 min timebox in step 2).
+   
+   If the support steward does not have the capacity, they should ping the **Engineering Manager** (or the delegated person) so he can secure resources to resolve that issue on the fly (see step 6 below).
```

Comment:
Mmm, quite interesting... I see your point. I would be towards having false positives to reduce toil, which means advocating to always give the support steward the green light if there is something critical.
Let me think about it a little bit more...

Reply:
> Having this as a choice still keeps some of the decision toil over the head of support stewards.

:+1: 

---

I favor Georgianas first option of "always prioritizing critical support tasks over any other planned task of a support steward" because I would find the alternative too stressful - leaving a "critical" task hanging while trying to focus on something else work related.

With that said, I'm now realize I'm thinking "critical" is something like an incident. Is "critical" something we use to mean "incident" really? If so, then we should instead rely on such procedure...

Hmmm I guess "critical" could refer to "this has to be done very very soon" and not be an incident describing something going wrong right now.

Tricky topic!

Reply:
My first intuition is that "critical" should trigger the "incident response" process unless it can be resolved really quickly. And follow [our definition of an incident](https://compass.2i2c.org/projects/managed-hubs/incidents/#what-counts-as-an-incident) to define a "critical" support ticket.

That said, I understand that in some cases, we might want a non-incident to be considered critical. For example if a community needs help with something that only affects a small number of people, but it's a really important community to help in that moment as defined by Partnerships.

Either way, we want "critical" to be a fairly high bar - if the bar is too low then everything will be labeled critical and we'll re-create our current pattern of reactiveness.

Reply:
> That said, I understand that in some cases, we might want a non-incident to be considered critical.

This is the main distinction when I am talking about something being "critical", I am not talking about incidents (which we know are critical by definition), I am talking about tasks that need to be done during the current sprint, on the fly, because of X reasons.

> Either way, we want "critical" to be a fairly high bar - if the bar is too low then everything will be labeled critical and we'll re-create our current pattern of reactiveness.

I totally concur, it should be an exception as I mentioned in the text.